spec/controllers/conversations_controller_spec.rb

```diff
@@ -0,0 +1,167 @@
+require 'rails_helper'
+
+RSpec.describe ConversationsController do
+  let(:valid_session) { {} }
+
+  describe 'GET #index' do
+    login_user
+    it 'renders the :index template' do
+      get :index, session: valid_session 
+      expect(response).to render_template :index
+      end
+  end
+
+  describe 'GET #show' do
+    it 'assign the requested conversation to @conversation' do
+      user = create(:user_with_conversations)
+      sign_in user
+      conversation = user.conversations.first
+      get :show, params: { id: conversation }
+      expect(assigns(:conversation)).to eq conversation
+    end
+
+    it "renders the :show template" do
+      user = create(:user_with_conversations)
+      sign_in user
+      conversation = user.conversations.first
+      get :show, params: { id: conversation }
+      expect(response).to render_template :show
+    end
+  end
```

Comment:
You could implement a `before :each` as you already implement it in messages_controller_spec.rb

Reply:
Thanks Goyo! i changed them for let (create...) :raised_hands: